### PR TITLE
132744 school info rename pages

### PR DESCRIPTION
**Pull request by danielryannimble**
pages names updated

page content updated

links updated

unit tests updated

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## 2.0.0
 
-* User Story 132316 : New 'Performance data' section added to task list.
```
Comment (dneed-nimble) [inline release-notes.md:3]: You've removed a line from the release notes, was that intended?

File: Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/TaskList/SchoolPupilForecasts/PupilForecasts.cshtml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿@page "/task-list/{id:int}/confirm-school-pupil-forecasts"
```
Comment (dneed-nimble) [inline Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/TaskList/SchoolPupilForecasts/PupilForecasts.cshtml:1]: Why is this changing? And in the future we are adding the ability to edit the pupil forecast I believe, should this not indicate that this is a summary page? Same applies for budget etc

Comment (dneed-nimble) [reply Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/TaskList/SchoolPupilForecasts/PupilForecasts.cshtml:1]: Will it truncate or replace to slug, i.e., are you expecting budget/update-budget?

File: release-notes.md (hunk 1 above)
Comment (danielryannimble) [reply release-notes.md:3]: ill put that back

**Review by dneed-nimble — APPROVED** [review]

**Pull request merged**